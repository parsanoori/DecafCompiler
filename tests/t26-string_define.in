define xyz zyx
define a b
xyz
xyzz
a
aa
"xyz"
"xyzz"
"a a"
"a"
"ab"