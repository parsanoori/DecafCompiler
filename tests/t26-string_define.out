T_ID zyx
T_ID xyzz
T_ID b
T_ID aa
T_STRINGLITERAL "zyx"
T_STRINGLITERAL "xyzz"
T_STRINGLITERAL "b b"
T_STRINGLITERAL "b"
T_STRINGLITERAL "ab"

